pages/careers/_open-positions.html

```diff
@@ -89,7 +93,77 @@ <h4>Help us a create a fundamentally better DevOps experience!</h4>
+    <div style="background-color:#2e3b4a; padding:4px 6px; border-radius: 10px; display:inline-block; font-size:14px;"><strong>Time Zone:</strong> GMT+0 to GMT+2</div>
+
+    <h4>What You'll Work On</h4>
+    <ul>
+      <li>
+        <strong>Build a brand new product.</strong>
+        We have a new, unannounced product in the works. Join a small team to help design, prototype, test, launch,
+        and develop this exciting new initiative that we believe will transform the software industry.
+      </li>
+      <li>
+        <strong>CLI platform.</strong>
+        Develop a part of this new product by building a beautiful, first-class CLI experience using Go.
+      </li>
+      <li>
+        <strong>Web platform.</strong>
+        Integrate your new product work with our platform, which consists of a REST API (Next.js/Typescript) and a
+        web-based single-page app (React/Typescript, Tailwind).
+      </li>
+      <li>
+        <strong>Infrastructure.</strong>
+        Deploy and manage the new product on AWS using tools such as Fargate, Lambda, API Gateway, EKS, and more.
+        Define and manage all of the infrastructure as code using tools such as Terraform, Docker, and Packer.
+      </li>
+      <li>
+        <strong>And a little bit of everything else.</strong>
+        Gruntwork is a
+        <a
+          href="https://blog.gruntwork.io/how-we-built-a-distributed-self-funded-family-friendly-profitable-startup-93635feb5ace"
+          target="_blank"
+          >small, distributed, self-funded, profitable startup</a
+        >, so we all wear a few hats. You should expect to write plenty of code,
+        but, depending on your interests, there will also be ample opportunity
+        to write blog posts, give talks, contribute to open source, go to conferences,
+        talk with customers, do sales calls, interview candidates, mentor new hires,
+        design products, come up with marketing ideas, discuss strategy,
+        and all the other tasks that are part of working at a small company.
+      </li>
+    </ul>
+    <h4>Your Ideal Background</h4>
+    <ul>
+      <li>You know how to write code across the stack and have experience in Golang.</li>
+      <li>Experience with Node/TypeScript is a plus.</li>
+      <li>Experience running production software, ideally with Terraform and AWS, is a plus.</li>
+      <li>You have a strong background in software engineering.</li>
+      <li>Your happy place is writing code, but you enjoy seeing it in action with real customers.</li>
+      <li>You are inspired by <a href="/about/#our-values">our values</a>.</li>
+      <li>Bonus points for a sense of humor, empathy, and curiosity.</li>
+    </ul>
+    <h4>Compatible time zones</h4>
+    <p>
+      This role is specifically for the <strong>GMT+0 (e.g., London) to GMT+2 (e.g., Berlin) time zones</strong>,
```

Comment:
I find using GMT as a basis a bit confusing, because of the daylight savings shift. E.g, right now Berlin is GMT+2, and London is GMT+1. Would it be better to use something like "Europe time zones" and "Continental US time zones".

Reply:
Good point. Agreed that GMT can be a bit confusing. On the flip side, "Europe time zones" is a bit too vague: e.g., we don't really hire in Eastern Europe. The advantage of GMT is it does make it more precise. Is there some version of this that's as precise, but easier to reason about and not affected by daylight savings?

Reply:
Trying out some options:

> This role is specifically for European time zones, with the western boundary being London's time zone, and the eastern boundary being Berlin's time zone, though you can be located in almost any city/country within those time zones.

Maybe something like that?

And then for the US, the same thing, except swapping SF for London and Boston for Berlin?

Reply:
Oh I thought we decided we are ok with hiring in Eastern Europe, but I may be wrong about that.

Anyway, your suggestion LGTM, Jim!

Reply:
Updated time zone wording: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/710/commits/8d59066ee32540df5b4f9ff9c143e5c73500e4d5

>Oh I thought we decided we are ok with hiring in Eastern Europe, but I may be wrong about that.

For contractors, yes, but for full-timers, probably not. With full-time folks, they will need overlap with our US teams—e.g., for all-hands meetings, occasionally cross-time zone product development, etc—so we should probably stay relatively strict with time zones there.